test/integration.spec.js

```diff
@@ -98,7 +98,7 @@ it('integration : babel syntax error', async() => {
     '',
     'error  in ./test/fixtures/babel-syntax/index.js',
     '',
-    `Syntax Error: Unexpected token (5:11)
+    `Unexpected token (5:11)
```

Comment:
this errror should be `Syntax Error: Unexpected token`, so you should not change this test

Reply:
reverted!